client/src/styles/Icons.css

```diff
@@ -0,0 +1,74 @@
+@import url('./index.css');
+.bookmark__icon.solid {
+    position: relative;
+    fill: royalblue;
+}
+
+.like__icon.solid {
+    position: relative;
+    fill: rgb(148, 65, 225);
+}
+
+.log__icon.solid {
+    position: relative;
+    fill: limegreen;
+}
```

Comment:
These styles belong in the respective files, `BkmarkButton.css`, `LikeButton.css`, and `LogButton.css`